code/script.js

```diff
@@ -3,17 +3,156 @@
 // Step 1 - Welcome and introduction
 // Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+  `Welcome to our Javascript Pizzeria.🍕 Ready to Start? - Click 'OK' to begin.`
+);
+
+let userName = prompt("Write your name in the box below 👇");
+alert("👋 Hello, " + userName + "! Click 'OK' to unlock the menu!");
+console.log("Hello, " + userName + "!");
```

Comment:
Please remove the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to 🧹